unit/av_test.c

```diff
@@ -1002,22 +1002,22 @@ int main(int argc, char **argv)
 	if (!hints)
 		return EXIT_FAILURE;
 
-	while ((op = getopt(argc, argv, "f:d:D:n:a:s:h")) != -1) {
+	while ((op = getopt(argc, argv, "p:g:G:n:f:s:h")) != -1) {
 		switch (op) {
-		case 'd':
+		case 'g':
 			good_address = optarg;
 			break;
-		case 'D':
+		case 'G':
```

Comment:
Can we use long options for specifying good and bad addresses? I think this is used only in this test.